enterprise/tpa_pipeline.py

```diff
@@ -142,11 +143,18 @@ def redirect_to_consent():
     if not enterprise_customer.requests_data_sharing_consent:
         # This enterprise customer attached to this pipeline element does not request data sharing consent;
         # proceed with the creation of a link between the user and the enterprise customer, then exit.
-        ec_user, _ = EnterpriseCustomerUser.objects.get_or_create(
+        ec_user, __ = EnterpriseCustomerUser.objects.get_or_create(
             enterprise_customer=enterprise_customer,
             user_id=user.id
         )
 
+        messages.success(backend.strategy.request, _('Thank you for creating an account with edX.'))
+        if not user.is_active:
+            messages.info(
+                backend.strategy.request,
```

Comment:
nit: We **have** just sent you **an** email ...  (representing an action that has been done in recently) .